ssg/build_remediations.py

```diff
@@ -304,6 +329,17 @@ def generate_platform_conditional(self, platform):
             pkg_check_command = PKG_MANAGER_TO_PACKAGE_CHECK_COMMAND[pkg_manager]
             return pkg_check_command.format(platform)
 
+    def generate_platform_variables(self, platform):
+        if platform.startswith("var_") or platform.startswith("not_var_"):
+            if not self.local_env_yaml:
+                return None
+
+            cpe = self.local_env_yaml["product_cpes"].get_cpe(platform)
+            assert cpe.variable
+            var_include = ". /usr/share/scap-security-guide/remediation_functions\n"
```

Comment:
This pseudo-include is something that @j-ode has removed recently and we don't want to introduce it again.

I think that you don't need it anymore and you can replace `populate` with the `bash_instantiate_variables` Jinja macro.

Reply:
thanks! I added a commit addressing this.